### DISCUSSION TITLE
Manage runners between github.com and self hosted GHE

### DISCUSSION THREAD

**Discussion by gtorre** [actions/actions-runner-controller#2137] [Questions]
👋

I see that using [arc across organizations](https://github.com/actions/actions-runner-controller/blob/master/docs/using-arc-across-organizations.md) is supported, but is this the case in a scenario where you have repos on github.com and some on an internal github appliance? If it is, can someone point me to an example?

Thanks,
Gennaro

**Comment by skoonin** [comment]
i am also wondering about this? or do you need two instances of ARC?
